romancal/source_catalog/source_catalog.py

```diff
@@ -445,6 +448,15 @@ def update_metadata(self):
             aper_radii["annulus_arcsec"] = aper_radii.pop("annulus").value
             self.meta["aperture_radii"] = aper_radii
 
+            if self.ee_spline:
+                fractions = []
+                for name in self.aperture_cat.fractions:
+                    fraction = getattr(self.aperture_cat, name)
+                    setattr(self, name, fraction)
+                    fractions.append(fraction)
+
+                self.meta["ee_fractions"] = np.array(fractions).astype(np.float32)
```

Comment:
Store the `ee_fractions` as an array corresponding `aperture_radii` entries' arrays.